platform/admin/ui/src/test/js/wd/InstallationTests.js

```diff
@@ -0,0 +1,269 @@
+        });
+
+        // Check for protocols
+        it('should contain http and https protocols', function() {
+            return this.browser
+                .waitForElementByClassName('dropdown-toggle', asserters.textInclude('https'), shared.timeout).click()
+                .waitForElementByCssSelector('input[value="http://"]').click()
+                .waitForElementByClassName('dropdown-toggle', !asserters.textInclude('https'), shared.timeout).click()
+                .waitForElementByCssSelector('input[value="https://"]').click()
+                .waitForElementByClassName('dropdown-toggle', asserters.textInclude('https'), shared.timeout);
+        });
+
+        // Check for host/port/site name/ organization
+        it('should contain host, port, site name and organization text fields', function() {
+            return this.browser
+                .waitForElementByName('host', shared.timeout)
+                .waitForElementByName('port', shared.timeout)
+                .waitForElementByName('id', shared.timeout)
+                .waitForElementByName('organization', shared.timeout);
+        });
+
+        it('should contain next button', function () {
+            return this.browser
+                .waitForElementById('nextStep', shared.timeout).click();
+        });
+    });
+
+    describe('configure anonymous claims attributes  page', function () {
+        it('should contain configure general settings', function () {
+            return this.browser
+                .waitForElementById('anonClaims', asserters.textInclude('Configure anonymous claims attributes'), shared.timeout);
+        });
+
+        it('should contain a profile dropdown', function() {
+            return this.browser
+                .waitForElementByCssSelector('button[title="Default"]', shared.timeout).click()
+                .waitForElementByClassName('dropdown-menu', asserters.isDisplayed, shared.timeout)
+                .waitForElementById('anonClaims', shared.timeout).click()
+        });
+
+        it('should have the default attributes', function() {
+            return this.browser
+                .waitForConditionInBrowser("document.getElementsByClassName('claim-table')[0].rows.length === 3", shared.timeout)
+                .waitForElementByCssSelector('input[value="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier"]', shared.timeout)
+                .waitForElementByCssSelector('input[value="http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role"]', shared.timeout);
+        });
+
+        it('should have defaults for dropdown', function() {
+            return this.browser
+                .waitForElementByClassName('fa-caret-down', shared.timeout).click()
+                .waitForElementByClassName('dropdown-menu', asserters.isDisplayed, shared.timeout)
+                .takeScreenshot().saveScreenshot(shared.getPathForScreenshot('attribute-dropdown.png'))
```

Comment:
Taking screenshots is slow so I would suggest removing these unless they are actually be used.